src/app/file/jpeg_format.cpp

```diff
@@ -177,12 +178,38 @@ bool JpegFormat::onLoad(FileOp* fop)
   // Start decompressor.
   jpeg_start_decompress(&dinfo);
 
+  // Get orientation:
+  TinyEXIF::EXIFInfo info;
+  int orientation = 0;
+#if LAF_WINDOWS
+  const int tempBufSize = (int)std::ftell(file);
+  rewind(file);
+  char *tempBuf = (char *)malloc(sizeof(char) * tempBufSize);
+  if (!tempBuf) {
+    std::free(tempBuf);
+    jpeg_destroy_decompress(&dinfo);
+    return false;
+  }
+  fread(tempBuf, sizeof(char), tempBufSize, file);
+  info.parseFrom((uint8_t*)tempBuf, tempBufSize);
+  orientation = info.Orientation;
+  std::free(tempBuf);
+#elif LAF_MACOS
+  info.parseFrom(file->_bf._base, file->_bf._size);
+  orientation = info.Orientation;
+#elif LAF_LINUX
+  const int fileSize = std::ftell(file);
+  info.parseFrom((uint8_t*)file->_IO_read_base, fileSize);
+  orientation = info.Orientation;
+#endif
```

Comment:
I think we can avoid using platform specific code if we put something like the following in line 144:
```c++
  int orientation = 0;
  {
    std::ifstream istream(fop->filename(), std::ifstream::in | std::ifstream::binary);
    // Get EXIF information:
    TinyEXIF::EXIFInfo info(istream);
    if (info.Fields)
      orientation = info.Orientation;
  }
```